config/puma.rb

```diff
@@ -23,7 +23,7 @@
 worker_timeout 3600 if ENV.fetch("RAILS_ENV", "development") == "development"
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
-port ENV.fetch("PORT") { 3000 }
+port ENV.fetch("PORT") { 5000 }
 
```

Comment:
Changed the port to 5000 so that we can run FE on 3000 and BE on 5000 with `rails s`